src/app/charts/chart.component.html

```diff
@@ -11,6 +11,12 @@ <h2 class="chart-label">
         </button>
     </div>
 </div>
+<div *ngIf="chart.indicator.thresholdIndicator" class="chart-options">
```

Comment:
How do you envision adding additional sets of extra params components here? Just as a long list of if blocks? I see adding additional extra params components getting unwieldy fairly quickly. There are a limited number to add though, so some significantly abstracted intermediate component might be overkill as well.